## Travis tests have failed

Hey @joshghent,
Please read the following log in order to understand the failure reason.
It'll be awesome if you fix what's wrong and commit the changes.


### Node.js: node

<a href="https:&#x2F;&#x2F;travis-ci.org&#x2F;esfiddle&#x2F;esfiddle&#x2F;jobs&#x2F;708558614">View build log</a>


<details>
  <summary>
    <strong>
     npm run test &amp;&amp; npm run cover
    </strong>
  </summary>

```
> es6-fiddle-web@1.0.0 test /home/travis/build/esfiddle/esfiddle
> grunt test && npm run lint-server && npm run test-node

Loading "less-lint-task.js" tasks...ERROR
>> TypeError: require(...).tmpDir is not a function
Warning: Task "lesslint" not found. Use --force to continue.

Aborted due to warnings.
npm ERR! code ELIFECYCLE
npm ERR! errno 3
npm ERR! es6-fiddle-web@1.0.0 test: `grunt test && npm run lint-server && npm run test-node`
npm ERR! Exit status 3
npm ERR! 
npm ERR! Failed at the es6-fiddle-web@1.0.0 test script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/travis/.npm/_logs/2020-07-16T01_55_23_716Z-debug.log
```

</details>


### Node.js: 6

<a href="https:&#x2F;&#x2F;travis-ci.org&#x2F;esfiddle&#x2F;esfiddle&#x2F;jobs&#x2F;708558616">View build log</a>


<details>
  <summary>
    <strong>
     npm run test &amp;&amp; npm run cover
    </strong>
  </summary>

```
> es6-fiddle-web@1.0.0 test /home/travis/build/esfiddle/esfiddle
> grunt test && npm run lint-server && npm run test-node

Running "lesslint:target" (lesslint) task
style/main.less (289)
Rule doesn't have all its properties in alphabetical order. Assure properties are in alphabetical order (order-alphabetical)
   style/partials/helpers.less [Line 1, Column 1]:	 .full-width {
   style/partials/helpers.less [Line 21, Column 1]:	 .select-bg {
   style/components/general.less [Line 4, Column 1]:	 body, html {
   style/components/general.less [Line 24, Column 1]:	 .CodeMirror {
   style/components/general.less [Line 35, Column 1]:	 .CodeMirror pre {
   style/components/general.less [Line 46, Column 1]:	 .CodeMirror-gutters {
   style/components/general.less [Line 51, Column 1]:	 .CodeMirror-linenumber {
   style/components/general.less [Line 74, Column 1]:	 .cm-fat-cursor .CodeMirror-cursor {
   style/components/general.less [Line 82, Column 1]:	 .cm-animate-fat-cursor {
   style/components/general.less [Line 110, Column 1]:	 .CodeMirror-rulers {
   style/components/general.less [Line 118, Column 1]:	 .CodeMirror-ruler {
   style/components/general.less [Line 226, Column 1]:	 .CodeMirror {
   style/components/general.less [Line 231, Column 1]:	 .CodeMirror-scroll {
   style/components/general.less [Line 244, Column 1]:	 .CodeMirror-sizer {
   style/components/general.less [Line 251, Column 1]:	 .CodeMirror-vscrollbar,
   style/components/general.less [Line 259, Column 1]:	 .CodeMirror-vscrollbar {
   style/components/general.less [Line 265, Column 1]:	 .CodeMirror-hscrollbar {
   style/components/general.less [Line 271, Column 1]:	 .CodeMirror-scrollbar-filler {
   style/components/general.less [Line 275, Column 1]:	 .CodeMirror-gutter-filler {
   style/components/general.less [Line 279, Column 1]:	 .CodeMirror-gutters {
   style/components/general.less [Line 286, Column 1]:	 .CodeMirror-gutter {
   style/components/general.less [Line 296, Column 1]:	 .CodeMirror-gutter-wrapper {
   style/components/general.less [Line 302, Column 1]:	 .CodeMirror-gutter-background {
   style/components/general.less [Line 308, Column 1]:	 .CodeMirror-gutter-elt {
   style/components/general.less [Line 323, Column 1]:	 .CodeMirror pre {
   style/components/general.less [Line 344, Column 1]:	 .CodeMirror-wrap pre {
   style/components/general.less [Line 349, Column 1]:	 .CodeMirror-linebackground {
   style/components/general.less [Line 357, Column 1]:	 .CodeMirror-linewidget {
   style/components/general.less [Line 366, Column 1]:	 .CodeMirror-scroll,
   style/components/general.less [Line 375, Column 1]:	 .CodeMirror-measure {
   style/components/general.less [Line 382, Column 1]:	 .CodeMirror-cursor {
   style/components/general.less [Line 389, Column 1]:	 div.CodeMirror-cursors {
   style/components/general.less [Line 532, Column 1]:	 .cm-s-3024-day .CodeMirror-matchingbracket {
   style/components/general.less [Line 623, Column 1]:	 .cm-s-3024-night .CodeMirror-matchingbracket {
   style/components/general.less [Line 826, Column 1]:	 .cm-s-ambiance.CodeMirror {
   style/components/general.less [Line 839, Column 1]:	 .cm-s-ambiance .CodeMirror-linenumber {
   style/components/general.less [Line 947, Column 1]:	 .cm-s-base16-dark .CodeMirror-matchingbracket {
   style/components/general.less [Line 1038, Column 1]:	 .cm-s-base16-light .CodeMirror-matchingbracket {
   style/components/general.less [Line 1110, Column 1]:	 .cm-s-bespin .CodeMirror-matchingbracket {
   style/components/general.less [Line 1207, Column 1]:	 .cm-s-blackboard .CodeMirror-matchingbracket {
   style/components/general.less [Line 1288, Column 1]:	 .cm-s-cobalt .CodeMirror-matchingbracket {
   style/components/general.less [Line 1380, Column 1]:	 .cm-s-dracula.CodeMirror,
   style/components/general.less [Line 1463, Column 1]:	 .cm-s-dracula .CodeMirror-matchingbracket {
   style/components/general.less [Line 1471, Column 1]:	 .cm-s-eclipse span.cm-keyword {
   style/components/general.less [Line 1533, Column 1]:	 .cm-s-eclipse .CodeMirror-matchingbracket {
   style/components/general.less [Line 1577, Column 1]:	 .cm-s-elegant .CodeMirror-matchingbracket {
   style/components/general.less [Line 1681, Column 1]:	 .cm-s-erlang-dark .CodeMirror-matchingbracket {
   style/components/general.less [Line 1753, Column 1]:	 .cm-s-hopscotch .CodeMirror-matchingbracket {
   style/components/general.less [Line 1764, Column 1]:	 .cm-s-icecoder {
   style/components/general.less [Line 1864, Column 1]:	 .cm-s-icecoder div.CodeMirror-selected {
   style/components/general.less [Line 1868, Column 1]:	 .cm-s-icecoder .CodeMirror-gutters {
   style/components/general.less [Line 1877, Column 1]:	 .cm-s-icecoder .CodeMirror-matchingbracket {
   style/components/general.less [Line 1952, Column 1]:	 .cm-s-isotope .CodeMirror-matchingbracket {
   style/components/general.less [Line 2082, Column 1]:	 .cm-s-lesser-dark .CodeMirror-matchingbracket {
   style/components/general.less [Line 2087, Column 1]:	 .cm-s-liquibyte.CodeMirror {
   style/components/general.less [Line 2222, Column 1]:	 .cm-s-liquibyte div.CodeMirror-simplescroll-vertical div {
   style/components/general.less [Line 2262, Column 1]:	 .cm-s-material .CodeMirror-gutters {
   style/components/general.less [Line 2351, Column 1]:	 .cm-s-material .cm-error {
   style/components/general.less [Line 2355, Column 1]:	 .cm-s-material .CodeMirror-matchingbracket {
   style/components/general.less [Line 2465, Column 1]:	 .cm-s-mdn-like.CodeMirror {
   style/components/general.less [Line 2570, Column 1]:	 div.cm-s-mdn-like span.CodeMirror-matchingbracket {
   style/components/general.less [Line 2594, Column 1]:	 .cm-s-midnight.CodeMirror {
   style/components/general.less [Line 2668, Column 1]:	 .cm-s-midnight .CodeMirror-matchingbracket {
   style/components/general.less [Line 2760, Column 1]:	 .cm-s-monokai .CodeMirror-matchingbracket {
   style/components/general.less [Line 2768, Column 1]:	 .cm-s-neat span.cm-keyword {
   style/components/general.less [Line 2776, Column 1]:	 .cm-s-neat span.cm-builtin {
   style/components/general.less [Line 2781, Column 1]:	 .cm-s-neat span.cm-special {
   style/components/general.less [Line 2802, Column 1]:	 .cm-s-neat .CodeMirror-matchingbracket {
   style/components/general.less [Line 2840, Column 1]:	 .cm-s-neo .CodeMirror-gutters {
   style/components/general.less [Line 2845, Column 1]:	 .cm-s-neo .CodeMirror-linenumber {
   style/components/general.less [Line 2855, Column 1]:	 .cm-s-neo .CodeMirror-cursor {
   style/components/general.less [Line 2939, Column 1]:	 .cm-s-night .CodeMirror-matchingbracket {
   style/components/general.less [Line 2949, Column 1]:	 .cm-s-panda-syntax {
   style/components/general.less [Line 2964, Column 1]:	 .cm-s-panda-syntax .cm-comment {
   style/components/general.less [Line 3013, Column 1]:	 .cm-s-panda-syntax .CodeMirror-matchingbracket {
   style/components/general.less [Line 3113, Column 1]:	 .cm-s-paraiso-dark .CodeMirror-matchingbracket {
   style/components/general.less [Line 3204, Column 1]:	 .cm-s-paraiso-light .CodeMirror-matchingbracket {
   style/components/general.less [Line 3378, Column 1]:	 .cm-s-railscasts .CodeMirror-matchingbracket {
   style/components/general.less [Line 3476, Column 1]:	 .cm-s-seti.CodeMirror {
   style/components/general.less [Line 3481, Column 1]:	 .cm-s-seti .CodeMirror-gutters {
   style/components/general.less [Line 3560, Column 1]:	 .cm-s-seti .CodeMirror-matchingbracket {
   style/components/general.less [Line 3622, Column 1]:	 .cm-s-solarized {
   style/components/general.less [Line 3627, Column 1]:	 .cm-s-solarized.cm-s-dark {
   style/components/general.less [Line 3707, Column 1]:	 .cm-s-solarized .cm-hr {
   style/components/general.less [Line 3727, Column 1]:	 .cm-s-solarized .cm-error,
   style/components/general.less [Line 3991, Column 1]:	 .cm-s-tomorrow-night-bright .CodeMirror-matchingbracket {
   style/components/general.less [Line 4082, Column 1]:	 .cm-s-tomorrow-night-eighties .CodeMirror-matchingbracket {
   style/components/general.less [Line 4356, Column 1]:	 .cm-s-twilight .CodeMirror-matchingbracket {
   style/components/general.less [Line 4455, Column 1]:	 .cm-s-vibrant-ink .CodeMirror-matchingbracket {
   style/components/general.less [Line 4567, Column 1]:	 .cm-s-xq-dark .CodeMirror-matchingbracket {
   style/components/general.less [Line 4594, Column 1]:	 .cm-s-xq-light span.cm-keyword {
   style/components/general.less [Line 4648, Column 1]:	 .cm-s-xq-light .CodeMirror-matchingbracket {
   style/components/general.less [Line 4662, Column 1]:	 .cm-s-yeti.CodeMirror {
   style/components/general.less [Line 4667, Column 1]:	 .cm-s-yeti .CodeMirror-gutters {
   style/components/general.less [Line 4823, Column 1]:	 .cm-s-zenburn span.CodeMirror-matchingbracket {
   style/components/general.less [Line 4828, Column 1]:	 .cm-s-zenburn span.CodeMirror-nonmatchingbracket {
   style/components/general.less [Line 4849, Column 1]:	 .CodeMirror-scroll {
   style/components/general.less [Line 4864, Column 1]:	 [data-balloon]:before,
   style/components/general.less [Line 4886, Column 1]:	 [data-balloon]:after {
   style/components/general.less [Line 4897, Column 1]:	 [data-balloon]:before {
   style/components/general.less [Line 4927, Column 1]:	 .login-page {
   style/components/typography.less [Line 1, Column 1]:	 h1,
   style/components/typography.less [Line 8, Column 1]:	 h2 {
   style/components/typography.less [Line 40, Column 1]:	 .about {
   style/components/buttons.less [Line 13, Column 1]:	 .share {
   style/components/buttons.less [Line 17, Column 5]:	 &:before {
   style/components/buttons.less [Line 13, Column 1]:	 .share {
   style/components/buttons.less [Line 13, Column 1]:	 .share {
   style/components/content.less [Line 1, Column 1]:	 .content {
   style/components/content.less [Line 17, Column 5]:	 &.column {
   style/components/content.less [Line 1, Column 1]:	 .content {
   style/components/content.less [Line 1, Column 1]:	 .content {
   style/components/content.less [Line 58, Column 5]:	 &.fiddle-wrapper {
   style/components/content.less [Line 58, Column 5]:	 &.fiddle-wrapper {
   style/components/content.less [Line 98, Column 5]:	 &.result-wrapper {
   style/components/content.less [Line 98, Column 5]:	 &.result-wrapper {
   style/components/header.less [Line 1, Column 1]:	 .header {
   style/components/header.less [Line 1, Column 1]:	 .header {
   style/components/header.less [Line 1, Column 1]:	 .header {
   style/components/header.less [Line 1, Column 1]:	 .header {
   style/components/select.less [Line 1, Column 1]:	 .selector-wrapper {
   style/components/settings.less [Line 1, Column 1]:	 .settings {
   style/components/settings.less [Line 10, Column 3]:	 & ~ .settings-toggle {
   style/components/settings.less [Line 21, Column 5]:	 & ~ .settings-popup {
   style/components/settings.less [Line 24, Column 7]:	 &:before {
   style/components/snackbar.less [Line 1, Column 1]:	 .snackbar {
   style/components/snackbar.less [Line 14, Column 3]:	 &.show {
   style/partials/media-queries.less [Line 2, Column 3]:	 .content .code-wrapper {
   style/partials/media-queries.less [Line 38, Column 3]:	 .main{
   style/components/banner.less [Line 1, Column 1]:	 .global-banner {
Element (div.CodeMirror-secondarycursor) is overqualified, just use .CodeMirror-secondarycursor without element name. Don't use classes or IDs with elements (a.foo or a#foo). (overqualified-elements)
   style/components/general.less [Line 71, Column 13]:	 div.CodeMirror-secondarycursor {
Use of !important Be careful when using !important declaration (important)
   style/components/general.less [Line 76, Column 3]:	 border: 0 !important;
   style/components/general.less [Line 232, Column 3]:	 overflow: scroll !important;
   style/components/general.less [Line 299, Column 3]:	 background: none !important;
   style/components/general.less [Line 300, Column 3]:	 border: none !important;
   style/components/general.less [Line 534, Column 3]:	 color: #a16a94 !important;
   style/components/general.less [Line 625, Column 3]:	 color: white !important;
   style/components/general.less [Line 949, Column 3]:	 color: white !important;
   style/components/general.less [Line 1040, Column 3]:	 color: white !important;
   style/components/general.less [Line 1057, Column 3]:	 background: #36312e !important;
   style/components/general.less [Line 1067, Column 3]:	 border-left: 1px solid #797977 !important;
   style/components/general.less [Line 1112, Column 3]:	 color: white !important;
   style/components/general.less [Line 1209, Column 3]:	 color: white !important;
   style/components/general.less [Line 1290, Column 3]:	 color: white !important;
   style/components/general.less [Line 1382, Column 3]:	 background-color: #282a36 !important;
   style/components/general.less [Line 1383, Column 3]:	 color: #f8f8f2 !important;
   style/components/general.less [Line 1465, Column 3]:	 color: white !important;
   style/components/general.less [Line 1535, Column 3]:	 color: black !important;
   style/components/general.less [Line 1579, Column 3]:	 color: black !important;
   style/components/general.less [Line 1683, Column 3]:	 color: white !important;
   style/components/general.less [Line 1700, Column 3]:	 background: #433b42 !important;
   style/components/general.less [Line 1710, Column 3]:	 border-left: 1px solid #989498 !important;
   style/components/general.less [Line 1755, Column 3]:	 color: white !important;
   style/components/general.less [Line 1878, Column 3]:	 color: #fff !important;
   style/components/general.less [Line 1879, Column 3]:	 background: #555 !important;
   style/components/general.less [Line 1899, Column 3]:	 background: #404040 !important;
   style/components/general.less [Line 1909, Column 3]:	 border-left: 1px solid #c0c0c0 !important;
   style/components/general.less [Line 1954, Column 3]:	 color: white !important;
   style/components/general.less [Line 2084, Column 3]:	 color: white !important;
   style/components/general.less [Line 2357, Column 3]:	 color: white !important;
   style/components/general.less [Line 2450, Column 3]:	 color: #ffb928 !important;
   style/components/general.less [Line 2584, Column 3]:	 background: #314D67 !important;
   style/components/general.less [Line 2670, Column 3]:	 color: white !important;
   style/components/general.less [Line 2762, Column 3]:	 color: white !important;
   style/components/general.less [Line 2804, Column 3]:	 color: black !important;
   style/components/general.less [Line 2941, Column 3]:	 color: white !important;
   style/components/general.less [Line 3115, Column 3]:	 color: white !important;
   style/components/general.less [Line 3206, Column 3]:	 color: white !important;
   style/components/general.less [Line 3307, Column 3]:	 color: #8F938F !important;
   style/components/general.less [Line 3325, Column 3]:	 background: #272935 !important;
   style/components/general.less [Line 3335, Column 3]:	 border-left: 1px solid #d4cfc9 !important;
   style/components/general.less [Line 3380, Column 3]:	 color: white !important;
   style/components/general.less [Line 3455, Column 3]:	 color: #F0F !important;
   style/components/general.less [Line 3477, Column 3]:	 background-color: #151718 !important;
   style/components/general.less [Line 3478, Column 3]:	 color: #CFD2D1 !important;
   style/components/general.less [Line 3562, Column 3]:	 color: white !important;
   style/components/general.less [Line 3993, Column 3]:	 color: white !important;
   style/components/general.less [Line 4084, Column 3]:	 color: white !important;
   style/components/general.less [Line 4358, Column 3]:	 color: white !important;
   style/components/general.less [Line 4457, Column 3]:	 color: white !important;
   style/components/general.less [Line 4569, Column 3]:	 color: white !important;
   style/components/general.less [Line 4650, Column 3]:	 color: black !important;
   style/components/general.less [Line 4663, Column 3]:	 background-color: #ECEAE8 !important;
   style/components/general.less [Line 4664, Column 3]:	 color: #d1c9c0 !important;
   style/components/general.less [Line 4757, Column 3]:	 background: #3f3f3f !important;
   style/partials/media-queries.less [Line 20, Column 5]:	 font-size: 1.4rem !important;
Fallback background (hex or RGB) should precede RGBA background. For older browsers that don't support RGBA, HSL, or HSLA, provide a fallback color. (fallback-colors)
   style/components/general.less [Line 218, Column 3]:	 background: rgba(255, 150, 0, 0.3);
   style/components/general.less [Line 556, Column 3]:	 background: rgba(58, 52, 50, 0.99);
   style/components/general.less [Line 561, Column 3]:	 background: rgba(58, 52, 50, 0.99);
   style/components/general.less [Line 638, Column 3]:	 background: rgba(56, 56, 56, 0.99);
   style/components/general.less [Line 643, Column 3]:	 background: rgba(56, 56, 56, 0.99);
   style/components/general.less [Line 810, Column 3]:	 background: rgba(255, 255, 255, 0.15);
   style/components/general.less [Line 813, Column 3]:	 background: rgba(255, 255, 255, 0.1);
   style/components/general.less [Line 818, Column 3]:	 background: rgba(255, 255, 255, 0.1);
   style/components/general.less [Line 823, Column 3]:	 background: rgba(255, 255, 255, 0.1);
   style/components/general.less [Line 854, Column 3]:	 background: none repeat scroll 0% 0% rgba(255, 255, 255, 0.031);
   style/components/general.less [Line 880, Column 3]:	 background: rgba(48, 48, 48, 0.99);
   style/components/general.less [Line 885, Column 3]:	 background: rgba(48, 48, 48, 0.99);
   style/components/general.less [Line 1129, Column 3]:	 background: rgba(37, 59, 118, 0.99);
   style/components/general.less [Line 1134, Column 3]:	 background: rgba(37, 59, 118, 0.99);
   style/components/general.less [Line 1222, Column 3]:	 background: rgba(179, 101, 57, 0.99);
   style/components/general.less [Line 1227, Column 3]:	 background: rgba(179, 101, 57, 0.99);
   style/components/general.less [Line 1366, Column 3]:	 background: rgba(255, 255, 255, 0.12);
   style/components/general.less [Line 1396, Column 3]:	 background: rgba(255, 255, 255, 0.1);
   style/components/general.less [Line 1401, Column 3]:	 background: rgba(255, 255, 255, 0.1);
   style/components/general.less [Line 1406, Column 3]:	 background: rgba(255, 255, 255, 0.1);
   style/components/general.less [Line 1461, Column 3]:	 background: rgba(255, 255, 255, 0.1);
   style/components/general.less [Line 1592, Column 3]:	 background: rgba(179, 101, 57, 0.99);
   style/components/general.less [Line 1597, Column 3]:	 background: rgba(179, 101, 57, 0.99);
   style/components/general.less [Line 1979, Column 3]:	 background: rgba(69, 68, 59, 0.99);
   style/components/general.less [Line 1984, Column 3]:	 background: rgba(69, 68, 59, 0.99);
   style/components/general.less [Line 2276, Column 3]:	 background: rgba(255, 255, 255, 0.15);
   style/components/general.less [Line 2279, Column 3]:	 background: rgba(255, 255, 255, 0.1);
   style/components/general.less [Line 2284, Column 3]:	 background: rgba(255, 255, 255, 0.1);
   style/components/general.less [Line 2289, Column 3]:	 background: rgba(255, 255, 255, 0.1);
   style/components/general.less [Line 2292, Column 3]:	 background: rgba(0, 0, 0, 0);
   style/components/general.less [Line 2375, Column 3]:	 background: rgba(113, 108, 98, 0.99);
   style/components/general.less [Line 2380, Column 3]:	 background: rgba(113, 108, 98, 0.99);
   style/components/general.less [Line 2453, Column 3]:	 background: rgba(255, 255, 255, 0.37);
   style/components/general.less [Line 2604, Column 3]:	 background: rgba(49, 77, 103, 0.99);
   style/components/general.less [Line 2609, Column 3]:	 background: rgba(49, 77, 103, 0.99);
   style/components/general.less [Line 2684, Column 3]:	 background: rgba(73, 72, 62, 0.99);
   style/components/general.less [Line 2689, Column 3]:	 background: rgba(73, 72, 62, 0.99);
   style/components/general.less [Line 2858, Column 3]:	 background: rgba(155, 157, 162, 0.37);
   style/components/general.less [Line 2873, Column 3]:	 background: rgba(68, 68, 119, 0.99);
   style/components/general.less [Line 2878, Column 3]:	 background: rgba(68, 68, 119, 0.99);
   style/components/general.less [Line 2959, Column 3]:	 background: rgba(99, 123, 156, 0.1);
   style/components/general.less [Line 3046, Column 3]:	 background: rgba(65, 50, 63, 0.99);
   style/components/general.less [Line 3051, Column 3]:	 background: rgba(65, 50, 63, 0.99);
   style/components/general.less [Line 3223, Column 3]:	 background: rgba(221, 240, 255, 0.2);
   style/components/general.less [Line 3228, Column 3]:	 background: rgba(221, 240, 255, 0.2);
   style/components/general.less [Line 3233, Column 3]:	 background: rgba(221, 240, 255, 0.2);
   style/components/general.less [Line 3303, Column 3]:	 background: rgba(255, 255, 255, 0.031);
   style/components/general.less [Line 3396, Column 3]:	 background: rgba(56, 86, 111, 0.99);
   style/components/general.less [Line 3401, Column 3]:	 background: rgba(56, 86, 111, 0.99);
   style/components/general.less [Line 3493, Column 3]:	 background: rgba(255, 255, 255, 0.1);
   style/components/general.less [Line 3498, Column 3]:	 background: rgba(255, 255, 255, 0.1);
   style/components/general.less [Line 3503, Column 3]:	 background: rgba(255, 255, 255, 0.1);
   style/components/general.less [Line 3736, Column 3]:	 background: rgba(7, 54, 66, 0.99);
   style/components/general.less [Line 3741, Column 3]:	 background: rgba(7, 54, 66, 0.99);
   style/components/general.less [Line 3818, Column 3]:	 background: rgba(255, 255, 255, 0.06);
   style/components/general.less [Line 3821, Column 3]:	 background: rgba(0, 0, 0, 0.06);
   style/components/general.less [Line 3834, Column 3]:	 background: rgba(45, 45, 45, 0.99);
   style/components/general.less [Line 3839, Column 3]:	 background: rgba(45, 45, 45, 0.99);
   style/components/general.less [Line 4015, Column 3]:	 background: rgba(45, 45, 45, 0.99);
   style/components/general.less [Line 4020, Column 3]:	 background: rgba(45, 45, 45, 0.99);
   style/components/general.less [Line 4255, Column 3]:	 background: rgba(50, 50, 50, 0.99);
   style/components/general.less [Line 4260, Column 3]:	 background: rgba(50, 50, 50, 0.99);
   style/components/general.less [Line 4372, Column 3]:	 background: rgba(53, 73, 60, 0.99);
   style/components/general.less [Line 4377, Column 3]:	 background: rgba(53, 73, 60, 0.99);
   style/components/general.less [Line 4492, Column 3]:	 background: rgba(39, 0, 122, 0.99);
   style/components/general.less [Line 4497, Column 3]:	 background: rgba(39, 0, 122, 0.99);
   style/components/general.less [Line 4887, Column 3]:	 background: rgba(17, 17, 17, 0.9);
Outlines should only be modified using :focus. Use of outline: none or outline: 0 should be limited to :focus rules. (outline-none)
   style/components/general.less [Line 231, Column 1]:	 .CodeMirror-scroll {
   style/components/general.less [Line 362, Column 1]:	 .CodeMirror-code {
height can't be used with display: inline. Certain properties shouldn't be used with certain display property values. (display-property-grouping)
   style/components/general.less [Line 288, Column 3]:	 height: 100%;
margin-bottom can't be used with display: inline. Certain properties shouldn't be used with certain display property values. (display-property-grouping)
   style/components/general.less [Line 291, Column 3]:	 margin-bottom: -30px;
Property with star prefix found. Checks for the star property hack (targets IE6/7) (star-property-hack)
   style/components/general.less [Line 293, Column 3]:	 *zoom: 1;
   style/components/general.less [Line 294, Column 3]:	 *display: inline;
   style/components/general.less [Line 425, Column 3]:	 *vertical-align: text-bottom;
Duplicate property '*display' found. Duplicate properties must appear one after the other. (duplicate-properties)
   style/components/general.less [Line 294, Column 3]:	 *display: inline;
The property -ms-user-select is compatible with -webkit-user-select and -moz-user-select and should be included as well. Include all compatible vendor prefixes to reach a wider range of users. (compatible-vendor-prefixes)
   style/components/general.less [Line 314, Column 3]:	 -webkit-user-select: none;
Element (div.CodeMirror-dragcursors) is overqualified, just use .CodeMirror-dragcursors without element name. Don't use classes or IDs with elements (a.foo or a#foo). (overqualified-elements)
   style/components/general.less [Line 394, Column 1]:	 div.CodeMirror-dragcursors {
Element (span.CodeMirror-selectedtext) is overqualified, just use .CodeMirror-selectedtext without element name. Don't use classes or IDs with elements (a.foo or a#foo). (overqualified-elements)
   style/components/general.less [Line 442, Column 1]:	 span.CodeMirror-selectedtext {
Values of 0 shouldn't have units specified. You don't need to specify units when a value is 0. (zero-units)
   style/components/general.less [Line 854, Column 3]:	 background: none repeat scroll 0% 0% rgba(255, 255, 255, 0.031);
   style/components/general.less [Line 854, Column 3]:	 background: none repeat scroll 0% 0% rgba(255, 255, 255, 0.031);
Fallback background-color (hex or RGB) should precede RGBA background-color. For older browsers that don't support RGBA, HSL, or HSLA, provide a fallback color. (fallback-colors)
   style/components/general.less [Line 2190, Column 3]:	 background-color: rgba(255, 0, 0, 0.25);
   style/components/general.less [Line 2193, Column 3]:	 background-color: rgba(255, 255, 255, 0.12);
   style/components/general.less [Line 2196, Column 3]:	 background-color: rgba(0, 255, 0, 0.15);
   style/components/general.less [Line 2208, Column 3]:	 background-color: rgba(150, 255, 0, 0.3);
   style/components/general.less [Line 2214, Column 3]:	 background-color: rgba(80, 80, 80, 0.7);
   style/components/general.less [Line 2218, Column 3]:	 background-color: rgba(80, 80, 80, 0.3);
Fallback border-left (hex or RGB) should precede RGBA border-left. For older browsers that don't support RGBA, HSL, or HSLA, provide a fallback color. (fallback-colors)
   style/components/general.less [Line 2484, Column 3]:	 border-left: 6px solid rgba(0, 83, 159, 0.65);
Fallback border-right-color (hex or RGB) should precede RGBA border-right-color. For older browsers that don't support RGBA, HSL, or HSLA, provide a fallback color. (fallback-colors)
   style/components/general.less [Line 3020, Column 3]:	 border-right-color: rgba(255, 255, 255, 0.1);
Fallback border (hex or RGB) should precede RGBA border. For older browsers that don't support RGBA, HSL, or HSLA, provide a fallback color. (fallback-colors)
   style/components/general.less [Line 3306, Column 3]:	 border: 1px solid rgba(255, 255, 255, 0.25);
Unqualified attribute selectors are known to be slow. (unqualified-attributes)
   style/components/general.less [Line 4860, Column 1]:	 [data-balloon] {
   style/components/general.less [Line 4864, Column 1]:	 [data-balloon]:before,
   style/components/general.less [Line 4865, Column 1]:	 [data-balloon]:after {
   style/components/general.less [Line 4886, Column 1]:	 [data-balloon]:after {
   style/components/general.less [Line 4897, Column 1]:	 [data-balloon]:before {
   style/components/general.less [Line 4906, Column 1]:	 [data-balloon]:hover:before,
   style/components/general.less [Line 4907, Column 1]:	 [data-balloon][data-balloon-visible]:before,
   style/components/general.less [Line 4907, Column 1]:	 [data-balloon][data-balloon-visible]:before,
   style/components/general.less [Line 4908, Column 1]:	 [data-balloon]:hover:after,
   style/components/general.less [Line 4909, Column 1]:	 [data-balloon][data-balloon-visible]:after {
   style/components/general.less [Line 4909, Column 1]:	 [data-balloon][data-balloon-visible]:after {
Expected (<filter-function-list> | none) but found 'alpha(opacity=0)'. Properties should be known (listed in CSS3 specification) or be a vendor-prefixed property. (known-properties)
   style/components/general.less [Line 4867, Column 3]:	 filter: alpha(opacity=0);
Expected (<filter-function-list> | none) but found 'alpha(opacity=100)'. Properties should be known (listed in CSS3 specification) or be a vendor-prefixed property. (known-properties)
   style/components/general.less [Line 4911, Column 3]:	 filter: alpha(opacity=100);
Too many !important declarations (55), try to use less than 10 to avoid specificity issues. Be careful when using !important declaration (important)

>> 289 lint issues in 1 file (0 errors, 289 warnings)

Running "eslint:target" (eslint) task
(node:5428) DeprecationWarning: The 'parserOptions.ecmaFeatures.experimentalObjectRestSpread' option is deprecated. Use 'parserOptions.ecmaVersion' instead. (found in "node_modules/eslint-config-airbnb-base/index.js")

Done.

> es6-fiddle-web@1.0.0 lint-server /home/travis/build/esfiddle/esfiddle
> ./node_modules/eslint/bin/eslint.js server/** test/**


/home/travis/build/esfiddle/esfiddle/server/app.js
  67:1  warning  Unexpected console statement  no-console

/home/travis/build/esfiddle/esfiddle/server/db/dbTest.js
  46:22  warning  Unexpected console statement  no-console
  47:17  warning  Unexpected console statement  no-console
  50:19  warning  Unexpected console statement  no-console
  51:17  warning  Unexpected console statement  no-console
  54:19  warning  Unexpected console statement  no-console
  55:17  warning  Unexpected console statement  no-console

/home/travis/build/esfiddle/esfiddle/server/db/mongoose.js
  11:14  error    Missing trailing comma                                                                     comma-dangle
  24:93  error    Unexpected string concatenation                                                            prefer-template
  27:10  error    Unexpected parentheses around single function argument having a body with no curly braces  arrow-parens
  27:19  warning  Unexpected console statement                                                               no-console

/home/travis/build/esfiddle/esfiddle/server/db/users.js
  22:36  warning  Unexpected unnamed function  func-names

/home/travis/build/esfiddle/esfiddle/test/passport-mock.js
  16:18  warning  Unexpected unnamed function  func-names

/home/travis/build/esfiddle/esfiddle/test/seedData.js
  20:15  warning  Unexpected console statement  no-console
  21:15  warning  Unexpected console statement  no-console
  41:15  warning  Unexpected console statement  no-console
  42:15  warning  Unexpected console statement  no-console

✖ 17 problems (3 errors, 14 warnings)
  3 errors, 0 warnings potentially fixable with the `--fix` option.

(node:5450) DeprecationWarning: The 'parserOptions.ecmaFeatures.experimentalObjectRestSpread' option is deprecated. Use 'parserOptions.ecmaVersion' instead. (found in "node_modules/eslint-config-airbnb-base/index.js")
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! es6-fiddle-web@1.0.0 lint-server: `./node_modules/eslint/bin/eslint.js server/** test/**`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the es6-fiddle-web@1.0.0 lint-server script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/travis/.npm/_logs/2020-07-16T01_55_33_934Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! es6-fiddle-web@1.0.0 test: `grunt test && npm run lint-server && npm run test-node`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the es6-fiddle-web@1.0.0 test script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/travis/.npm/_logs/2020-07-16T01_55_33_958Z-debug.log
```

</details>


###### TravisBuddy Request Identifier: 72350fc0-c707-11ea-aa06-e17841301e13

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -20,7 +20,7 @@ services:
       - VIRTUAL_HOST=esfiddle.ghent.cloud,esfiddle.net
       - PORT=3001
       - VIRTUAL_PORT=3001
-      - LETSENCRYPT_HOST=esfiddle.ghent.cloud
+      - LETSENCRYPT_HOST=esfiddle.ghent.cloud,esfiddle.net
       - LETSENCRYPT_EMAIL=josh@ghent.cloud
     ports:
       - "3001:3001"

